.azure-ci/docker_scripts.sh

```diff
@@ -29,7 +29,7 @@ cd /io
 pip install -e ".[tests, doc]"
 
 # Test dev install with pytest and flake8
-pytest --cov gtda --cov-report xml
+pytest gtda --cov --cov-report xml
```

Comment:
This is just done for consistency with other parts of the azure CI, order does not actually matter.